wdl2cwl/main.py

```diff
@@ -151,7 +144,7 @@ def get_literal_name(
     # if the literal expr is used inside WDL.Expr.Apply
     # the literal value is what's needed
     parent = expr.parent  # type: ignore[union-attr]
-    if isinstance(parent, (WDL.Expr.Apply, WDL.Expr.IfThenElse)):
+    if isinstance(parent, (WDL.Expr.Apply, WDL.Expr.IfThenElse, WDL.Expr.Array)):
```

Comment:
For literals that are declared in an Array. This is useful when all the elements in an array are been iterated to be reproduced in a string joined by a coma.